src/SonarScanner.MSBuild.Shim/PropertiesFileGenerator.cs

```diff
@@ -89,7 +89,7 @@ public ProjectInfoAnalysisResult GenerateFile()
         var projectPropertiesPath = Path.Combine(analysisConfig.SonarOutputDir, ProjectPropertiesFileName);
         var result = new ProjectInfoAnalysisResult();
         var writer = new PropertiesWriter(analysisConfig, logger);
-        logger.LogDebug(Resources.MSG_GeneratingProjectProperties, projectPropertiesPath, SonarProduct.GetSonarProductToLog(analysisConfig.SonarQubeHostUrl));
+        logger.LogDebug(Resources.MSG_GeneratingProjectProperties, projectPropertiesPath, SonarProduct.LogName(analysisConfig.SonarQubeHostUrl));
```

Comment:
Do we have `HostInfo` in scope here? It would be better to derive the log name from `HostInfo.IsSonarCloud` if possible.

Reply:
Unfortunatelly no :/ That lives only in `ProcessedArgs` of the `PreProcessor` step.

I would need to store it to `SonarQubeAnalysisConfig.xml` file and use that value instead.